clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/view/EcsLoadBalancerProvider.java

```diff
@@ -158,7 +158,7 @@ public Item get(String name) {
             serviceList.add(service.getServiceName());
             targetGroupToServicesMap.put(tgArn, serviceList);
           } else {
-            Set<String> srcServices = Collections.singleton(service.getServiceName());
+            Set<String> srcServices = new HashSet<>(Arrays.asList(service.getServiceName()));
```

Comment:
You could also do something like this `Set<String> srcServices = Sets.newHashSet<>(service.getServiceName());`